scripts/aws_deploy.py

```diff
@@ -26,14 +26,14 @@ def deploy():
     )
     hash = output.stdout.decode("utf-8").strip()
     print("\ngithash: " + hash)
-    identifier = f"{hash}-{os.environ['NODE_VERSION']}"
-    distdir = identifier + "-dist"
+    identifier = os.environ["IDENTIFIER"]
+    distdir = os.environ["DISTDIR"]
```

Comment:
Can you clue me into what's going on here? it's not clear to me how we got to new `IDENTIFIER` you created in  deploy.yml from the old `f"{hash}-{os.environ['NODE_VERSION']}"`

Reply:
If you look at L42-46 in deploy.yml, there's a new step called "Setup vars". It uses a [Github Actions command to set a step's output](https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-an-output-parameter) which lets me use that output as `env` vars in the following steps of deploy.yml. It's not the cleanest looking setup but it's the only thing that would work for what I wanted to do.
 https://github.com/facebookincubator/RapiD/blob/19bfb98032d247b0c5767b34175d5e0bc84dd591/.github/workflows/deploy.yml#L42-L46